### PR TITLE
Add tags to BO/DC Asset bulk edit

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/lib/mixins/models.py [hunk 1]
```diff
@@ -88,8 +89,15 @@ def formfield(self, form_class=TaggitTagField, **kwargs):
         return super().formfield(form_class, **kwargs)
 
 
+class RalphTaggableManager(_TaggableManager):
```
Comment (mkurek) [inline src/ralph/lib/mixins/models.py]: why is this necessary?

File: src/ralph/lib/mixins/fields.py [hunk 2]
```diff
@@ -205,4 +208,8 @@ def has_changed(self, initial, data):
             data = [i.strip() for i in data.split(',') if i.strip()]
             changed = len(initial) != len(data) or set(initial) - set(data)
             return changed
+        elif not initial and data:
```
Comment (mkurek) [inline src/ralph/lib/mixins/fields.py]: you could simplify these conditions into one

File: src/ralph/lib/mixins/models.py (hunk 1 above)
Comment (szok) [inline src/ralph/lib/mixins/models.py]: Yes, because Django Model manager need have self.name, default TaggitManager has not this variable. 

https://github.com/alex/django-taggit/blob/develop/taggit/managers.py#L111
https://github.com/django/django/blob/1.8.12/django/db/models/manager.py#L71

If we not add self.name we get exception:

```
AttributeError at /back_office/backofficeasset/
'_TaggableManager' object has no attribute 'name'
```

File: src/ralph/lib/mixins/fields.py [hunk 3]
```diff
@@ -205,4 +208,4 @@ def has_changed(self, initial, data):
             data = [i.strip() for i in data.split(',') if i.strip()]
             changed = len(initial) != len(data) or set(initial) - set(data)
             return changed
-        return False
+        return initial or data
```
Comment (mkurek) [inline src/ralph/lib/mixins/fields.py:211]: :+1:

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:
